assignment4/assignmentFour.js

```diff
@@ -0,0 +1,48 @@
+$(document).ready(function() {
+    $('#action').click(function() {
+
+        var letterGrade = $('#letter-grade').val().toUpperCase();
+        var myFavoriteThings = [];
+            myFavoriteThings.push("Cats");   
+            myFavoriteThings.push("Shopping"); 
+            myFavoriteThings.push("Naps"); 
+            myFavoriteThings.push("Vacations"); 
+
+		//Write your decision structure between this comment...
+
+		//I can't get .write to work, I get a TypeError in my console saying it's not a function. 
+		//Since it wasn't in the lecture videos, and my research couldn't find anything to allow 
+		//me to use it, I went with .text 		
```

Comment:
Where did you see a reference to .write()? I would have expected .text() here. You've done a nice job!